tests/e2e/performance/test_pvc_creation_performance.py

```diff
@@ -137,8 +221,15 @@ def test_multiple_pvc_creation_measurement_performance(self, teardown_factory):
         Returns:
 
         """
-        number_of_pvcs = 120
-        log.info("Start creating new 120 PVCs")
+        number_of_pvcs = 12
+        log.info(f"Start creating new {number_of_pvcs} PVCs")
+
+        initial_count = self.pvc_count_validation(
+            storgeclass=self.sc_obj.name,
+            namespace=defaults.ROOK_CLUSTER_NAMESPACE,
+            test_state="start",
+        )
+        initial_count["to_create"] = number_of_pvcs
```

Comment:
If we getting the initial_count from pvc_count_validation then why there is an assignment in line number 232 ?

Reply:
since this dict is sent tp the function as **kwargs, and we add this addition key to the dict to be used in the function.